src/models/user.js

```diff
@@ -0,0 +1,59 @@
+const mongoose = require('mongoose');
+const bcrypt = require('bcryptjs');
+const jwt = require('jsonwebtoken');
+
+const UserSchema = new mongoose.Schema({
+    firstName: {
+        type: String,
+        trim: true,
+        required: [true, 'Please provide first name'],
+        minlength: 1,
+        maxlength: 50,
+    },
+    lastName: {
+        type: String,
+        trim: true,
+        required: [true, 'Please provide last name'],
+        minlength: 1,
+        maxlength: 50,
+    },
+    email: {
+        type: String,
```

Comment:
lets add lowercase to ensure that all of our emails are in the same format.

Reply:
I'll add lowercase to the emails